tests/resources/mir_datasets/billboard/McGill-Billboard/0035/full.lab

```diff
@@ -0,0 +1,180 @@
+0.0	0.859138321	N
```

Comment:
Just curious: are you testing different things in the different tracks and that's why you need more than one?

Reply:
I personally find it good practice to test things on more than 1 datapoint, just in case.